v2/src/plugins/markdownVariables.json

```diff
@@ -0,0 +1,7 @@
+{
+    "exactfolderrecipename": {
```

Comment:
This is just an example right? Perhaps, instead, put a real example and modify the sign out page across all recipes to use this new plugin instead.

Reply:
Yeah this was meant as a sample, yeah Ill modify the sign out pages instead that makes more sense